src/ui.tsx

```diff
@@ -1190,7 +1190,10 @@ function DragTarget() {
   </div>;
 }
 
-ReactDOM.render(<Root />, document.getElementById("app"));
+const container = document.getElementById("app");
+// eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+const root = createRoot(container!);
+root.render(<Root />);
```

Comment:
I prefer the point-free style most of the time, so I'll nitpick this to
```suggestion
// eslint-disable-next-line @typescript-eslint/no-non-null-assertion
createRoot(document.getElementById("app")!).render(<Root />);
```

Reply:
Fine with me, I was just following React docs.